### PR TITLE
Enchant names

### PR DESCRIPTION
**Pull request by mart-r**
Added possibility to show enchantment names instead of "Enchanted Book" in sign.

Added config entry to enabled this (disabled by default).

### CODE REVIEW

**Comment by mart-r** [comment]
> For all configuration changes, we should add update code in Quickshop#updateConfig for previous config, and change config version.

Good to know. I will change/add that later tonight.

There's also an issue where stuff like mending will show "Mending I" instead of just "Mending". I'll also address that (note to self: should be able to check the max level and if it's 1, don't include the level).

After I address the above, do you see any issues with regards to including this within the next version of your plugin?

**Comment by mart-r** [comment]
Are we good now?

File: src/main/java/org/maxgamer/quickshop/QuickShop.java [hunk 1]
```diff
@@ -1574,6 +1574,11 @@ private void updateConfig(int selectedVersion) {
             getConfig().set("config-version", 106);
             selectedVersion = 106;
         }
+        if (selectedVersion == 106) {
+            getConfig().set("use-enchantment-for-enchanted-book", false);
```
Comment (sandtechnology) [inline src/main/java/org/maxgamer/quickshop/QuickShop.java:1578]: You should use "shop.use-enchantment-for-enchanted-book" :D

Comment (mart-r) [reply src/main/java/org/maxgamer/quickshop/QuickShop.java]: Yep. Totally correct.

Also fixed the 'Mending I' type of issues.

**Comment by sandtechnology** [comment]
Thanks for contributing!

**Pull request merged**